api/lib/database/helpers.js

```diff
@@ -36,16 +36,16 @@ function escapeValue(value) {
       })
       return JSON.stringify(clone)
     } else {
-      return escape(value)
+      return `"${escape(value)}"`
     }
   } catch (error) {
```

Comment:
in which case do we need to catch?

Reply:
When JSON.parse throws the error of X token in invalid position (because it is not an stringified JSON and therefore errors in the check)

Reply:
As an alternative, I would not pass in stringified JSON but just pass in the object and stringify here, what do you think?

Reply:
Sounds like a much better way to do it, yes